Options/WeakAurasOptions.lua

```diff
@@ -7126,13 +7126,21 @@ function WeakAuras.CreateFrame()
         if not(combinedText == "") then
           combinedText = combinedText.."\n\n";
         end
-        combinedText = combinedText.."--"..childId.."\n";
+
+        local formatString
```

Comment:
I know that this makes the translators job harder, so maybe we should choose a different message... You decide.